voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt

```diff
@@ -137,6 +152,11 @@ class VotingActivity : AppCompatActivity() {
                 "proper JSON in its message field: ${block.transaction["message"]}."
         }
 
+        // Parse date field
+        val regex = Regex("^(.*?)GMT")
+        var strippedDate = regex.find(block.timestamp.toString())?.value.toString()
+        strippedDate = strippedDate.substring(0, strippedDate.length - 3)
```

Comment:
Consider using [DateFormat](https://developer.android.com/reference/java/text/DateFormat) to format dates, this looks very fragile

Reply:
This is definitely better, I've also implemented this change in the `blockListAdapter`.